### PR TITLE
Removed black space at top of screen on Android 5.x closes #3592

### LINKED ISSUE #3592: Blank space appearing above action bar
With changes from https://github.com/getlantern/lantern/issues/3419, particularly with LanternMainActivity extending AppCompatActivity instead of Activity now, we're seeing a blank space at the top of the screen. This only appears to be affecting Android 5.x

### PR DESCRIPTION
**Pull request by oxtoacart**

### CODE REVIEW

File: src/github.com/getlantern/lantern-mobile/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -15,11 +15,6 @@
         <item name="color">#39C2D6</item>
     </style>
 
-    <style name="Theme.HomeScreen" parent="android:Theme.Holo.NoActionBar.TranslucentDecor">
```
Comment (oxtoacart) [inline src/github.com/getlantern/lantern-mobile/app/src/main/res/values/styles.xml]: This wasn't necessary for the fix, but appeared to be dead code so I removed it.

File: src/github.com/getlantern/lantern-mobile/app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```
Comment (oxtoacart) [inline src/github.com/getlantern/lantern-mobile/app/src/main/res/values/styles.xml]: This is the actual fix.

File: src/github.com/getlantern/lantern-mobile/app/src/main/res/values/styles.xml (hunk 1 above)
Comment (atavism) [inline src/github.com/getlantern/lantern-mobile/app/src/main/res/values/styles.xml]: Thanks! Yeah, this isn't being used anymore.

File: src/github.com/getlantern/lantern-mobile/app/src/main/res/values/styles.xml (hunk 2 above)
Comment (atavism) [inline src/github.com/getlantern/lantern-mobile/app/src/main/res/values/styles.xml]: Nice! It looks like ``AppTheme`` is used by default whenever our main activity extends ``AppCompatActivity`` but only with 5.x -- anyway, just verified the black status bar is gone now! Merging.

**Pull request merged**